shared/utils/crypto.py

```
@@ -41,6 +41,25 @@ def _get_encryption_key():
     return _aes_key, _aes_iv
 
 
+def _parse_aes_key(key: str) -> bytes:
+    """Parse raw, base64, or base64-prefixed AES key strings."""
+
+    if key.startswith("base64:"):
+        return base64.b64decode(key[7:], validate=True)
+
+    try:
+        return base64.b64decode(key, validate=True)
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)